recipe/meta.yaml

```
@@ -44,7 +51,7 @@ requirements:
     - filelock
     - pytorch =*=cuda*
     - lit
-    - cudatoolkit
+    - cuda-version {{ cuda_compiler_version }}  # [cuda_major >= 12]
```

Do we want to relax this pin at runtime somehow? Maybe like this?

```suggestion
    - cuda-version >={{ cuda_compiler_version }}  # [cuda_major >= 12]
```